CMakeLists.txt

```diff
@@ -33,6 +33,10 @@ nanobind_add_module(
   ${ISL_SOURCES}
 )
 
+set_source_files_properties(src/wrapper/wrap_isl_part1.cpp PROPERTIES COMPILE_FLAGS -Wno-deprecated-declarations)
+set_source_files_properties(src/wrapper/wrap_isl_part2.cpp PROPERTIES COMPILE_FLAGS -Wno-deprecated-declarations)
+set_source_files_properties(src/wrapper/wrap_isl_part3.cpp PROPERTIES COMPILE_FLAGS -Wno-deprecated-declarations)
```

Comment:
Why? They're legitimate warnings...

Reply:
They are legitimate, but I thought they are not very helpful since islpy also wraps deprecated isl functions, which is the reason for these warnings. This can also "hide" more serious warnings.

I don't feel particularly strongly in either direction here, but wanted to bring it up.